src/updatechecker.cpp

```diff
@@ -244,8 +242,7 @@ void UpdateChecker::Run()
 				}
 			}
 		}
-		// Check every 5 minutes
-		Sleep(300000);
+		Sleep(win_sparkle_get_update_check_interval());
```

Comment:
This is unfortunate, because it prevents the thread from being signalled in any way, e.g. to force manual check (which could then use the same thread) or to terminate. `m_terminateEvent.WaitUntilSignaled(timeout)` should be used instead.